src/app/recurring-deposits/recurring-deposits-account-view/recurring-deposits-account-view.component.html

```diff
@@ -0,0 +1,202 @@
+              Current Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.accountBalance}}<br />
+              Available Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.availableBalance}}<br />
+            </span>
+          </p>
+        </mat-card-subtitle>
+      </div>
+
+    </mat-card-title-group>
+
+    <mat-card-actions class="account-actions">
+
+      <ng-container *ngFor="let button of buttonConfig.singleButtons">
+        <button mat-raised-button >
+          <i class="{{button.icon}}"></i> {{button.name}}</button>
+      </ng-container>
+  
+      <ng-container *ngIf="buttonConfig.options.length > 0">
+        <button mat-raised-button [matMenuTriggerFor]="More">More</button>
+        <mat-menu #More="matMenu">
+        <span *ngFor="let option of buttonConfig.options">
+          <button mat-menu-item >{{option.name}}</button>
+        </span>
+        </mat-menu>
+      </ng-container>
+
+    </mat-card-actions>
+
+  </mat-card-header>
+
+  <mat-card-content class="content">
+
+    <div class="recurringDeposits-account-tables" fxLayout="row" fxLayoutGap="2%">
+
+      <div fxFlex="49%"
+        *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+        <h4 class="table-headers">Savings Details</h4>
+        <table>
+          <tbody>
+            <tr>
+              <td>Field Officer</td>
+              <td>{{recurringDepositsAccountData.fieldOfficerName}}<span
+                  *ngIf="!recurringDepositsAccountData.fieldOfficerName">Unassigned</span></td>
+            </tr>
+            <tr>
+              <td>Activation date</td>
+              <td>{{recurringDepositsAccountData.timeline.activatedOnDate | date}}</td>
+            </tr>
+            <tr>
+              <td>Maturity Date</td>
+              <td><span>{{recurringDepositsAccountData.maturityDate | date}}</span></td>
+            </tr>
+            <tr
+              *ngIf="recurringDepositsAccountData.timeline.closedOnDate || recurringDepositsAccountData.summary.totalWitddrawals">
+              <td *ngIf="recurringDepositsAccountData.timeline.closedOnDate">
+                Closed on Date</td>
+              <td><span>{{recurringDepositsAccountData.timeline.closedOnDate | date}}</span></td>
+            </tr>
+            <tr>
+              <td>Period</td>
+              <td>
+                <span>{{recurringDepositsAccountData.depositPeriod}}&nbsp;{{recurringDepositsAccountData.depositPeriodFrequency.value}}</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Deposit Frequency</td>
+              <td><span>{{recurringDepositsAccountData.recurringFrequency}}
+                  &nbsp;{{recurringDepositsAccountData.recurringFrequencyType.code}}
```

Comment:
see your screenshot for this

Reply:
As the web-app doesn't support translation for now. I have to choose the path of implementing through functions. And that's the reason that function has been written but presently that is giving some errors. Will modify it in the upcoming PR.

Reply:
use .value instead of .code